tests/flow/unicode/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -73,8 +73,6 @@ function utf16Length(str, pos) {
 exports[`test keys.js 1`] = `
 "({\'この事はつもり素晴らしいことさ\': \'35jL9V\'})
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-({
-  \"この事はつもり素晴らしいことさ\": \"35jL9V\"
-});
+({ \"この事はつもり素晴らしいことさ\": \"35jL9V\" });
```

Comment:
Flow has a bug where utf8 characters are not correctly measured and it messes up all the positions in the ast. In this case, `hasLineInRange` checks for basically the entire file. It used to incorrectly return yes because there is a new line at the end but now that we do trimRight, it's not there anymore.